build.gradle

```diff
@@ -135,7 +146,9 @@ subprojects {
             compile deps.log4j
             compile deps.guice
             compile deps.slf4j
-
+            // See: https://github.com/rzwitserloot/lombok/issues/1301
```

Comment:
Does it mean that any version of error_prone_core newer than 2.0.15 will also work?

Reply:
versions higher than 2.0.15 did not work and resulted in this issue. The suggested fix was to set to 2.0.15 version.

Reply:
Not able to upgrade is still an issue.
According to https://github.com/rzwitserloot/lombok/issues/1332 this issue is fixed. Could you give it a try?
Also after you merge in the latest change, please use one way to specify the error-prone version.

Reply:
With the latest version of error-prone, this issue has gone away.